pysqldb3/README.md

```diff
@@ -1 +1,1027 @@
+ - **`shp_name` str, default None**: Output filename to be used for shapefile (should end in .shp)
+ - **`cmd` str, default None**: GDAL command to overwrite default behavior 
+ - **`port` str, default None**:  Database port to use, defaults database connection port
+ - **`gdal_data_loc` str, default None**: Path to gdal data, if not stored in system env correctly
+ - **`precision` bool, default False**:  Sets precision flag in ogr (defaults to `-lco precision=NO`)
+ - **`private` bool, default False**: Flag for permissions output table in database (Defaults to False - will grant select to public)
+ - **`temp` bool, default True**: Optional flag to make table as not-temporary (defaults to True)
+ - **`shp_encoding` str, default None**: If not None, sets the PG client encoding while uploading the shpfile. Options inlude `LATIN1` or `UTF-8`.
+ - **`print_cmd` str, default None**: Option to print ogr command (without password)
+ - **`days` int, default 7**: Defines the lifespan (number of days) of any tables created in the query, before they are automatically deleted  
+ 
+**Sample**
+
+```
+>>> from pysqldb3 import pysqldb3
+>>> db = pysqldb3.DbConnect(type='pg', server=server_address, database='ris', user='shostetter', password='*******')
+>>> db.shp_to_table(path=r'C:\Users\HShi\Documents', shp_name='lion_sample_shp.shp', table='lion_sample_import', schema='working')
+
+b'0...10...20...30...40...50...60...70...80...90...100 - done.\r\n'
+- Query run 2023-01-16 16:37:57.418900
+ Query time: Query run in 12907 microseconds
+ * Returned 0 rows *
+
+>>> db.dfquery("select * from working.lion_sample_import limit 3")
+
+      ogc_fid segmentid number_tra  ...  created_dt created_tm                                               geom
+0        1   0294281          2  ...  2022-10-14   10:12:40  0105000020D70800000100000001020000000200000000...
+1        2   0043911          2  ...  2022-10-14   10:12:40  0105000020D70800000100000001020000000200000000...
+2        3   0118892          1  ...  2022-10-14   10:12:40  0105000020D70800000100000001020000000200000080...
+
+[3 rows x 8 columns]
+
+```
+
+[Back to Table of Contents](#pysqldb3-public-functions)
+<br>
+
+
+
+#### feature_class_to_table
+
+**`DbConnect.feature_class_to_table(path, table, schema=None, shp_name=None, gdal_data_loc=GDAL_DATA_LOC,
+                               private=False, temp=True, fc_encoding=None, print_cmd=False,
+                               days=7)`**
+
+Imports feature class from ESRI file geodatabase, uses GDAL to generate the table.
+
+###### Parameters:
+ - **`path` str**: File path of the geodatabase
+ - **`table` str**: Table name to be used in the database
+ - **`schema` str, default None**:  Database schema to use for destination in database (defaults database object's default schema)
+ - **`shp_name` str, default None**: Output filename to be used for shapefile
```

Comment:
this doesnt make sense, since there is no output shp. This must be an input name